addons/im_livechat/static/src/core/web/livechat_channel_tag_edit.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="im_livechat.ChannelTagEdit">
+        <div class="o-livechat-ChannelTagEdit d-flex flex-column p-2 flex-grow-1 bg-inherit">
+            <input class="o-livechat-ChannelTagEdit-search form-control lh-1 px-2 bg-view shadow-sm" t-att-value="state.searchStr" t-ref="input" placeholder="Search for channel tags" t-on-input="onInput"/>
```

Comment:
This looks like a valid use case for `t-model` instead of `t-att-value` + `t-on-input`